FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -18,9 +20,33 @@
 from DIRAC.Core.Security.X509Chain import X509Chain
 import DIRAC
 from DIRAC.Core.Utilities import InstallTools
+from DIRAC.FrameworkSystem.Client.ComponentMonitoringClient import ComponentMonitoringClient
+from DIRAC.Core.Utilities.ThreadScheduler import gThreadScheduler
+from DIRAC.FrameworkSystem.Client.SystemAdministratorClient import SystemAdministratorClient
 
 class SystemAdministratorHandler( RequestHandler ):
 
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    """
+    Handler class initialization
+    """
+    hostMonitoring = False
+
+    # Check the flag for monitoring of the state of the host
+    result = gConfig.getOption( 'DIRAC/HostMonitoring' )
```

Comment:
I cant come up with a better place to read the flag, as we only want to do this and creating the task once.
Im OK with making the flag default to True.